tests/integration_tests/performance/test_process_startup_time.py

```diff
@@ -16,29 +16,38 @@
 
 
 def test_startup_time_new_pid_ns(test_microvm_with_api):
-    """Check startup time when jailer is spawned in a new PID namespace."""
+    """
+    Check startup time when jailer is spawned in a new PID namespace.
+
+    @type: performance
+    """
     microvm = test_microvm_with_api
     microvm.bin_cloner_path = None
     microvm.jailer.new_pid_ns = True
-    _test_startup_time(microvm)
+    return _test_startup_time(microvm)
```

Comment:
What is the use of the "return"?

Reply:
The return values are caught by the test report and used to populate the json with the result and pass criteria of tests